05_syntax_lab/01.py

```diff
@@ -1,5 +1,6 @@
-"""
-Write a program that reads 10 numbers from
-the user and prints the largest one
-"""
-
+print 'Enter 10 integers, 1 in each line:'

```

Comment:
Hi Nir and welcome to the course,

Please leave the top comments in the file (the ones inside """). They make it easier for everyone opening the file later to understand what's inside